docs/pages/blog/introducing-pigment-css.md

```diff
@@ -78,19 +85,49 @@ Check out [How Pigment CSS works](https://github.com/mui/pigment-css/blob/maste
+
+| Metrics                      |                                                                                                        Emotion |                                                                                                        Pigment CSS | Change |
+| :--------------------------- | -------------------------------------------------------------------------------------------------------------: | -----------------------------------------------------------------------------------------------------------------: | -----: |
+| First Load JS                |                                                                                                          [131 kB](https://emotion-demo.vercel.app/?metric=1) |                                                                                                              [104 kB](https://pigment-css-demo.vercel.app/?metric=1) |   -20% |
+| Total blocking time          | [280 ms](https://pagespeed.web.dev/analysis/https-emotion-demo-vercel-app/uxfpkvbp31?hl=fr&form_factor=mobile) | [210 ms](https://pagespeed.web.dev/analysis/https-pigment-css-demo-vercel-app/azcw1qxkec?hl=fr&form_factor=mobile) |   -25% |
+
+:::info
+**Reproduction**. Because you should only trust numbers that you can reproduce, we've shared below how we found these numbers.
+
+- Emotion
+  - Live: https://emotion-demo.vercel.app/?metric=1
+  - Source: https://github.com/brijeshb42/emotion-demo
+- Pigment CSS
+  - Live: https://pigment-css-demo.vercel.app/?metric=1
+  - Source: https://github.com/brijeshb42/pigment-css-landing-page-demo
+
+:::
+
+### Better runtime performance
 
-When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, we've observed the following page load performance gains:
+When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, we've observed the following runtime performance gains using the same code, but powered by different style engines:
 
-| Metrics                | Emotion | Pigment CSS | Reduction |
-| :--------------------- | ------: | ----------: | --------: |
-| First load JavaScript  |   131kB |       104kB |       20% |
-| First Contentful Paint |   503ms |       455ms |        9% |
-| Time To First Byte     | 447.5ms |     381.5ms |       15% |
-| Total Page HTML        |  15.9kB |      14.7kB |      7.5% |
+<!-- After page load, runtime performance is also improved with Pigment CSS. -->
+
+| Metrics                                 | Emotion | Pigment CSS | Change |
+| :-------------------------------------- | ------: | ----------: | -----: |
+| Create and mount a new button           |  17.3 ms |      10.1 ms |   -42% |
+| Change a variant on a mounted component |  14.0 ms |      9.13 ms |   -34% |
+| Change a value inside a CSS prop        |  13.6 ms |      8.63 ms |   -37% |
```

Comment:
It's not that significant. https://stitches.dev/docs/benchmarks claims almost more. I would want to see how Linaria performs, CSS Modules, Tailwind CSS. I suspect we can get more performance out of this approach.

I have removed the other perf tests as we were not faster, similar to https://github.com/brijeshb42/pigment-css-bench where I couldn't really see performance improvements.

It will be key to validate that it's possible. Now, from my test in https://github.com/oliviertassinari/react-conf-2024-pigment-css/tree/master/benchmark, the performance gain was significant during the runtime: https://react-conf-2024-pigment-css.netlify.app/?slideIndex=34&stepIndex=0.